doc/onboarding-extras.md

```diff
@@ -78,6 +78,29 @@ Please use these when possible / appropriate
   git checkout $(git show -s --pretty='%T' $(git show-ref -d $(git describe --abbrev=0) | tail -n1 | awk '{print $1}')) -- test; make -j4 test
   ```
 
+### LTS/Version labels
+
+We use labels to keep track of which branches a commit should land on:
+
+* `dont-land-on-v?.x`
+  * For changes that do not apply to a certain release line
+  * Also used when the work of backporting a change outweighs the benefits
+* `land-on-v?.x`
+  * Used by releasers to mark a PR as scheduled for inclusion in an LTS release
+  * Applied to the original PR for clean cherry-picks, to the backport PR otherwise
+* `backport-requested-v?.x`
+  * Used to indicate that a PR needs a manual backport to a branch in order to land on it
```

Comment:
`in order to land it on it`? Although I'd be more explicit without anaphoras and say "in order to land the PR on that branch" or something like that. 

We might want to indicate this is typically because the commit doesn't land cleanly on the branch although that's implied.